### PR TITLE
fix(dp): Add carrier aggregation fields to cbsd write fields

### PR DESCRIPTION
**Pull request by WojSad**
Signed-off-by: Wojciech Sadowy <wojciech.sadowy@freedomfi.com>

fix(dp): Add carrier aggregation fields to cbsd write fields

### CODE REVIEW

File: dp/cloud/go/services/dp/builders/builders.go [hunk 1]
```diff
@@ -205,15 +205,6 @@ func (b *DBCbsdBuilder) WithCbsdCategory(cat string) *DBCbsdBuilder {
 	return b
 }
 
-// func (b *DBCbsdBuilder) WithDefaulValues() *DBCbsdBuilder {
```
Comment (jkmar) [inline dp/cloud/go/services/dp/builders/builders.go:208]: How did it got merged in the first place?

File: dp/cloud/go/services/dp/storage/cbsd_manager_test.go [hunk 2]
```diff
@@ -105,6 +105,20 @@ func (s *CbsdManagerTestSuite) TestCreateCbsdWithDefaultValues() {
...
+		b.NewDBCbsdBuilder().
+			WithCarrierAggregationEnabled(true).
+			WithGrantRedundancy(true).
+			WithMaxIbwMhx(200).Cbsd, registered))
+	s.Require().NoError(err)
+	s.verifyCbsdCreation(b.NewDBCbsdBuilder().
+		WithIndoorDeployment(false).
+		WithCarrierAggregationEnabled(true).
+		WithGrantRedundancy(true).
+		WithMaxIbwMhx(200).Cbsd)
```
Comment (jkmar) [inline dp/cloud/go/services/dp/storage/cbsd_manager_test.go]: 200 is too big, max should be 150

File: dp/cloud/go/services/dp/storage/cbsd_manager_test.go [hunk 3]
```diff
@@ -205,6 +219,9 @@ func (s *CbsdManagerTestSuite) TestUpdateCbsd() {
 		WithNumberOfPorts(cbsdBuilder.Cbsd.NumberOfPorts.Int64+4).
 		WithSingleStepEnabled(true).
 		WithIndoorDeployment(true).
+		WithCarrierAggregationEnabled(true).
```
Comment (jkmar) [inline dp/cloud/go/services/dp/storage/cbsd_manager_test.go:222]: How did this pass before?

**Review by jkmar — APPROVED** [review]

File: dp/cloud/go/services/dp/storage/cbsd_manager_test.go (hunk 3 above)
Comment (WojSad) [reply dp/cloud/go/services/dp/storage/cbsd_manager_test.go:222]: it defaulted to false

**Pull request merged**